rpc/rpcserver/server.go

```diff
@@ -1398,13 +1398,12 @@ func (t *ticketbuyerServer) StartAutoBuyer(ctx context.Context, req *pb.StartAut
 	}
 	params := wallet.ChainParams()
 
-	var votingAddress string
+	var votingAddress dcrutil.Address
 	if req.VotingAddress != "" {
-		votingAddr, err := decodeAddress(req.VotingAddress, params)
+		votingAddress, err = decodeAddress(req.VotingAddress, params)
 		if err != nil {
 			return nil, err
 		}
-		votingAddress = votingAddr.String()
 	}
 
 	var poolAddress string
```

Comment:
This variable isn't needed anymore.  We should still keep the decodeAddress call below to verify the input but there's no need to reconstruct the address string from the decoded address.